src/olympia/stats/migrations/0005_create_switch_for_fenix_build_ids.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 2.2.13 on 2020-07-27 08:43
+
+from django.db import migrations
+
+
+def create_waffle_switch(apps, schema_editor):
+    Switch = apps.get_model("waffle", "Switch")
+
+    Switch.objects.create(
+        name="use-fenix-build-ids",
```

Comment:
It doesn't hurt, but why the switch ?  It's not a big feature we're launching and/or liable to break is it?

Reply:
The switch is needed because I am working on the data as well. For now, we didn't identify a better value than the build ID (which seemed to be documented as the "app version" similar to what we have for FF Desktop). If we find a better app version for Fenix, then we'll update the BigQuery data and we'll need to disable this UI change on AMO.

Having the switch lets us work in parallel and won't require an AMO push to revert the small change. In the end, we'd like to have app versions for Fenix that look like Desktop (so `78.0`, `79.1`, etc.)